### DISCUSSION TITLE
New behaviour: pygubu doesn't higlight selected widgets or containers in the designer-panel

### DISCUSSION THREAD

**Discussion by Hook-shot** [alejandroautalan/pygubu-designer#223] [General]
I tried to get rid of all pygubu settings by proceeding uninstall-install, but the program allways starts with the same black theme, which I set before.
Whatever I try to change: elements, which I select in the left pane, will not be selected in the designer.
About three weeks ago everything worked fine.

**Comment by alejandroautalan** [comment]
Hello @Hook-shot thanks for trying pygubu.

Can you show me the command you used to install pygubu-designer?
What OS are you using?

Regards

**Comment by Hook-shot** [comment]
Am 10.10.2023 14:51 schrieb Alejandro Autalán ***@***.***>:
Hello @Hook-shot thanks for trying pygubu.
Can you show me the command you used to install pygubu-designer?
What OS are you using?
Regards

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
pip install pygubupip uninstall pygubuI solved the problem by uninstalling python and all Python extensions in vs code. After installing pygubu now I got clean installation.In the former surrounding I always got a ttkbootstrap theme collection, which I didn't get rid of.
